### ISSUE TITLE
Change Ammo Count for Clan Arena

### ISSUE THREAD

**Issue by thistlew** [deurk/ktx#39]
My friends and I were playing CA KTX last night, and we wanted to give ourselves more ammo (back when we used to play, the game was more of a spamfest). Anyway, we realized that we couldn't alter the ammo count, so we were wondering if altering the ammo count could be made a command in game (maybe admin only)?  Thanks!!

**Comment by tcsabina** [comment]
@dusty-qw ,

Does this issue make any sense to you? You are into CA...
Is this still a problem?
Do we want to solve this?

I know that this is a very old one, but would be nice to tackle it. Either close it as 'won't fix' or 'fixed', or leave it open till your next Pull Request ;).


Toma

**Comment by dusty-qw** [comment]
Yeah, the ability to customize all that stuff is definitely on my to-do list :) I'm not sure it's a problem but it would be a nice feature to have for sure.

**Comment by tcsabina** [comment]
Hi @dusty-qw,
Look what I have just found ;).

Is this still on the to-do list? ;)

**Comment by dusty-qw** [comment]
yes this is definitely coming soon, along with a bunch of other little changes :)
